code-studio/src/js/plc/header.jsx

```diff
@@ -0,0 +1,42 @@
+/*
+  Header used by PLC, has navigation breadcrumb
+ */
+
+import React from 'react';
+import color from '../color';
+
+var PlcHeader = React.createClass({
+  propTypes: {
+    unit_name: React.PropTypes.string,
+    course_view_path: React.PropTypes.string
+  },
+
+  render: function () {
+    let breadcrumbContainerStyle = {
+      marginBottom: '20'
+    };
+
+    let breadcrumbIconStyle = {
+      margin: 5,
+      color: color.charcoal
+    };
+
+    let currentPageStyle = {
+      color: color.orange
+    };
```

Comment:
Our [React style guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react) says to put all css styles in single `styles` object.